mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -1065,6 +1084,7 @@ pub async fn nexus_create(
     size: u64,
     uuid: Option<&str>,
     children: &[String],
```

Comment:
I thought we wanted *new* method nexus_create_v2(). This will also then allow us to fix more buried issues we have with the nexus in terms of adding locks?

Reply:
Done, for now it only has an extra `NexusNvmeParams` parameter.